packages/@ember/-internals/routing/lib/services/routing.ts

```diff
@@ -54,8 +54,10 @@ export default class RoutingService extends Service {
 
   generateURL(routeName: string, models: {}[], queryParams: {}) {
     let router = this.router;
-    // return early when the router microlib is not present, which is the case for {{link-to}} in integration tests
-    if (!router._routerMicrolib) {
+    // return early when the router microlib is not present, which is the case for <LinkTo/> in integration tests
```

Comment:
This part of the comment should be removed, it is no longer correct

Reply:
Good catch! Removed